middlewares/assignTask.js

```diff
@@ -4,7 +4,7 @@ const tasks = firestore.collection("tasks");
 
 const assignTask = async function (req, res) {
   try {
-    const { task } = await fetchSkillLevelTask("frontend", 1);
+    const { task } = await fetchSkillLevelTask("FRONTEND", 1);
```

Comment:
Can you please explain why are we passing skill `FRONTEND` by default?

Reply:
Also, I think it will be better to import it to from constants

Reply:
it is hard coded for now, it will come from the frontend when we have skills in the user object

Reply:
Got it. As discussed, please add a comment in code for now